controllers/storageclaim_controller.go

```diff
@@ -398,15 +404,33 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 				// storageclaim is clusterscoped resources using its
 				// hash as the clusterID
 				data["clusterID"] = r.storageClaimHash
+				labels["ramendr.openshift.io/storageID"] = storageID
 				if resource.Name == "cephfs" {
-					volumeSnapshotClass = r.getCephFSVolumeSnapshotClass(data)
+					volumeSnapshotClass = r.getCephFSVolumeSnapshotClass(data, labels)
 				} else if resource.Name == "ceph-rbd" {
-					volumeSnapshotClass = r.getCephRBDVolumeSnapshotClass(data)
+					volumeSnapshotClass = r.getCephRBDVolumeSnapshotClass(data, labels)
 				}
 				utils.AddAnnotation(volumeSnapshotClass, storageClaimAnnotation, r.storageClaim.Name)
 				if err := r.createOrReplaceVolumeSnapshotClass(volumeSnapshotClass); err != nil {
 					return reconcile.Result{}, fmt.Errorf("failed to create or update VolumeSnapshotClass: %s", err)
 				}
+			case "VolumeGroupSnapshotClass":
+				var volumeGroupSnapshotClass *groupsnapapi.VolumeGroupSnapshotClass
+				data["csi.storage.k8s.io/snapshotter-secret-namespace"] = r.OperatorNamespace
+				// generate a new clusterID for cephfs subvolumegroup, as
+				// storageclaim is clusterscoped resources using its
+				// hash as the clusterID
```

Comment:
infact you could delete the existing comment rather than copying it.

Reply:
Why? It is still relevant.